client/my-sites/media/main.jsx

```diff
@@ -119,6 +119,11 @@ export default React.createClass( {
 	},
 
 	getModalButtons() {
+		// do not render buttons if the media image editor is opened
+		if ( this.state.editedItem !== null ) {
+			return;
```

Comment:
we should pass `null` I guess.
https://github.com/Automattic/wp-calypso/blob/master/client/components/dialog/dialog-base.jsx#L46